### ISSUE TITLE
Different assignments for different groups of students?

### ISSUE THREAD

**Issue by kthyng** [jupyter/nbgrader#539]
I have a stacked class, so I have undergrads and grads in my class, and they have overlapping responsibilities. In a given assignment, the grad students have to do all of the problems, and the undergrads have to do most but not all of the problems. I saw in issue #488 that I could assign an additional hw or maybe have a separate notebook for the extra problems that only the grad students do, but then is there a way to have two student groups in my config that I assign a little differently? Or maybe that is a different way that would be better? Just wondering what my options here might be. Thanks.

**Comment by kthyng** [comment]
Maybe I should add that since sometimes the graduate students do an additional part on an existing undergrad problem, it would be best if their extra problems could be in the same notebook/problem set, and maybe I could just grade the undergrads out of a smaller number of points or something. Thanks again for your help!

**Comment by kthyng** [comment]
Ah, sorry to keep posting, but I have now realized I should probably just create two separate classes, and have the assignments in the undergrad class be copied from the grad class and then be edited down. I haven't tried it yet, but this seems totally doable! I'll close this issue assuming that will work, but open to more input if anyone has any.

**Comment by jhamrick** [comment]
Yes, I would probably recommend doing it as separate classes. This is not really a use case I'd considered before, but it's something worth thinking about further in the future, so I'm going to reopen this and put it on the wishlist at least :smile:

**Comment by jhamrick** [comment]
Yes, I would probably recommend doing it as separate classes. This is not really a use case I'd considered before, but it's something worth thinking about further in the future, so I'm going to reopen this and put it on the wishlist at least :smile:
